### DISCUSSION TITLE
Missing Belgian channels

### DISCUSSION THREAD

**Discussion by bert1111** [iptv-org/community#622] [Q&A]
I am trying to find a good playlist for Belgian tv streaming but so far can't find much. I am new to this, just started with experimenting today.
I used the playlist on this github but am missing much channels. Is there a reason that most of the channels are missing? I only see vtm2 and vtm and am missing the other channels on following link(some of them are payed). The other vtm channels and Play, VRT and Ketnet. https://www.prd.base.be/nl/tv/?CMP=sbr_base%20tv_e%7c&s_kwcid=AL!14365!3!701718871916!e!!g!!base%20tv&gad_source=1&gclid=EAIaIQobChMIwbLnuI6JigMVpjgGAB3y5gYcEAAYASABEgJecfD_BwE&gclsrc=aw.ds

**Comment by wim63** [comment]
Hello i am new here but i am belgian too and i am being suprised that flemish channels are not allowed to be seen by anyone unless payement is done. it would handy to find the links in m3u so that i dont need to pay for it because i am not allowed to pay for tv channels anyway cant afford it. I woul love to see the urls updated even though they are drm protected, they need to be somewhere available anyways. ii recently had a fallout in my network and due to a long fallout network time i missed a programme i cant rewatch so i would llike it to be able to be found somewhere in the www so lastly i have been fiddeling around with the links in the kodi plugin for vrtmax (named plugin.vrt.nu) but because of needed login i only got black screen but it seemed to be streaming in the vlc player. only the link should be in m3u8 and not in .ts that is the problem for now. not being able to stream the links. i wanted to as someone to look into it to make those links work again without needed to buy it anywways. great thanks in advance if someone has them and can provide them here. Kind Regards wim

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
